src/azure-cli/azure/cli/command_modules/appconfig/custom.py

```diff
@@ -39,28 +53,102 @@ def show_configstore(cmd, client, name, resource_group_name=None):
     return client.get(resource_group_name, name)
 
 
-def configstore_update_get():
-    return ConfigurationStoreUpdateParameters()
-
-
-def configstore_update_set(cmd,
-                           client,
-                           parameters,
-                           name,
-                           resource_group_name=None):
+def update_configstore(cmd,
+                       client,
+                       name,
+                       resource_group_name=None,
+                       tags=None,
+                       sku=None):
     if resource_group_name is None:
         resource_group_name, _ = resolve_resource_group(cmd, name)
 
-    update_params = ConfigurationStoreUpdateParameters(tags=parameters.tags)
+    update_params = ConfigurationStoreUpdateParameters(tags=tags,
+                                                       sku=sku)
+
     return client.update(resource_group_name=resource_group_name,
                          config_store_name=name,
                          config_store_update_parameters=update_params)
 
 
-def configstore_update_custom(instance, tags=None):
-    if tags is not None:
-        instance.tags = tags
-    return instance
+def assign_managed_identity(cmd, client, name, resource_group_name=None, identities=None):
+    if resource_group_name is None:
+        resource_group_name, _ = resolve_resource_group(cmd, name)
+
+    if identities is None:
+        identities = [SYSTEM_ASSIGNED_IDENTITY]
+
+    if '[all]' in identities:
```

Comment:
why not check this in validator?

Reply:
The parameter --identities can be used in these two commands  ```az appconfig identity assign``` and ```az appconfig identity remove``` and they use the same argument validator. However, '[all]' is only supported in remove. So either we check here or use two separate but very similar validators for these two commands. 

---
In reply to: [366210641](https://github.com/Azure/azure-cli/pull/11795#discussion_r366210641) [](ancestors = 366210641)

Reply:
add check in validator and will remove the check here. 

---
In reply to: [366530549](https://github.com/Azure/azure-cli/pull/11795#discussion_r366530549) [](ancestors = 366530549,366210641)